pkg/security/probe/probe.go

```diff
@@ -1043,6 +1045,13 @@ func NewProbe(config *config.Config, client *statsd.Client) (*Probe, error) {
 	return p, nil
 }
 
+func (p *Probe) sanityChecks() {
```

Comment:
It seems like this could have a better name?  "Check" suggests that this is checking something, not changing something.  And "Sanity" is extremely vague.

Reply:
Yeah the idea behind this was to be similar to the module `sanityChecks` were this was before. I will change to something more clear